inc/commonitilobject.class.php

```diff
@@ -7290,10 +7290,21 @@ function showTimeline($rand) {
             echo "<form id='validationanswers_id_{$item_i['id']}' class='center' action='$form_url' method='post'>";
             echo Html::hidden('id', ['value' => $item_i['id']]);
             echo Html::hidden('users_id_validate', ['value' => $item_i['users_id_validate']]);
-            Html::textarea([
-               'name'   => 'comment_validation',
-               'rows'   => 5
-            ]);
+
+            $rand       = mt_rand();
+            $content_id = "content$rand";
+            $cols    = 100;
+            $rows    = 10;
+
+            Html::textarea(['name'              => 'comment_validation',
```

Comment:
CS is not ideal here:
pass to the line after opening square

Reply:
done